sys/include/net/gnrc/netif/hdr.h

```diff
@@ -29,6 +29,7 @@
 #include "net/gnrc/pkt.h"
 #include "net/gnrc/pktbuf.h"
 #include "net/gnrc/netif.h"
+#include "timex.h"
```

Comment:
timex is for some lightweight `struct tm` alternative. I'm pretty sure this is not needed here. If you need e.g. `US_PER_MS` or so, include `time_units.h` instead.